two.js

```diff
@@ -0,0 +1,135 @@
+// Elliott Katane - 6214821
+
+let myForm = document.getElementById('myform');
+
+myForm.addEventListener('submit', function(e) {
+
+    let inputNom = document.getElementById('nomform');
+    let regexNom = /^[a-zA-Z-\s]{3,15}$/; 
+
+    if (inputNom.value.trim() == "") {;
```

Comment:
Personal preference but usually there exist methods called `isEmpty()` on String to check whether they are equal to "" and I think it makes the code a bit easier to read when using those,